<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>篮球联赛个人技术数据处理系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
    <layout class="QHBoxLayout" name="mainHorizontalLayout">
     <item>
      <widget class="QWidget" name="leftPanel">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="leftButtonLayout">
         <item>
          <widget class="QPushButton" name="newGameButton">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>48</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>13</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>添加新比赛</string>
            </property>
            <property name="objectName">
             <string>btnPrimary</string>
            </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="importDataButton">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>48</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>13</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>导入数据</string>
            </property>
            <property name="objectName">
             <string>btnGray</string>
            </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="exportDataButton">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>48</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>13</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>导出数据</string>
            </property>
            <property name="objectName">
             <string>btnGreen</string>
            </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
          </spacer>
         </item>
        </layout>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="summaryTab">
         <attribute name="title">
          <string>数据汇总</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
            <widget class="QTableWidget" name="summaryTableWidget"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="gamesTab">
         <attribute name="title">
          <string>比赛记录</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
            <widget class="QTableWidget" name="gamesTableWidget"/>
          </item>
         </layout>
        </widget>
      </widget>
     </item>
    </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
   </widget>
   <widget class="QMenu" name="menuQuery">
    <property name="title">
     <string>查询</string>
    </property>
    <addaction name="actionShowSummary"/>
    <addaction name="actionTopThree"/>
    <addaction name="actionTeamTopThree"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuQuery"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>保存数据</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>加载数据</string>
   </property>
  </action>
  <action name="actionShowSummary">
   <property name="text">
    <string>查看汇总数据</string>
   </property>
  </action>
  <action name="actionTopThree">
   <property name="text">
    <string>查看平均数据排名</string>
   </property>
  </action>
  <action name="actionTeamTopThree">
   <property name="text">
    <string>查看队内排名</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
